lib/buffer.js

```diff
@@ -70,6 +70,11 @@ function Buffer(arg, encoding) {
 Object.setPrototypeOf(Buffer.prototype, Uint8Array.prototype);
 Object.setPrototypeOf(Buffer, Uint8Array);
 
+if (Symbol.species && Buffer[Symbol.species] === Buffer) {
```

Comment:
If species ends up being staged, but not shipped, for a Node release, do you think it might make sense to keep the conditional?